src/message/messagesActions.js

```diff
@@ -47,5 +50,5 @@ export const messageLinkPress = (href: string) => (dispatch: Dispatch, getState:
     dispatch(doNarrow(narrow, anchor));
     return;
   }
-  openLink(getFullUrl(href, auth.realm));
+  openLink(await getFileDownloadUrl(href, auth));
```

Comment:
nit in commit message (and the one before it): use a consistent form for "this fixes issue #NNN", [namely](https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#github-prs-issues)
> When a commit fixes an issue, use a line like `Fixes: #1234` at the end of the commit message.

(The most important bit there is that we've standardized on one magic word, "fixes" rather than "closes" or their other friends. But the same form beyond that is helpful too.)

Reply:
Changed to Fixes.

Reply:
OK but please use the particular form we've standardized on, as described in the bit of our style guide which I quoted :slightly_smiling_face:

Reply:
Thanks for the revision but it says `Fixes: #3033.` -- our standard form is `Fixes: #3033` :slightly_smiling_face: